ledger/cow.go

```diff
@@ -204,6 +213,10 @@ func (cb *roundCowState) put(addr basics.Address, new basics.AccountData, newCre
 	}
 }
 
+func (cb *roundCowState) trackCreatable(creatable *basics.CreatableLocator) {
```

Comment:
I think I'd rather see this take it's argument by value, since it ends up copying anyway.

Reply:
Good idea